### PR TITLE
fix build and fix unit tests to work again

### PR DESCRIPTION
**Pull request by skybrian**
- Jetbrains warning about unnecessary type inference seems to be incorrect?
- Registering the same action id twice causes tests to fail

### CODE REVIEW

**Comment by skybrian** [comment]
Probably not the right fix; suggestions welcome. I'd be interested in knowing if it also failed on your workstation.

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -236,12 +236,12 @@
...
-              icon="FlutterIcons.HotReload">
-        <keyboard-shortcut keymap="$default" first-keystroke="ctrl alt SEMICOLON"/>
-        <keyboard-shortcut keymap="$default" first-keystroke="ctrl BACK_SLASH"/>
-      </action>
+      <!--<action id="Flutter.Toolbar.ReloadAction" class="io.flutter.actions.ReloadFlutterAppRetarget"-->
+              <!--description="Reload"-->
+              <!--icon="FlutterIcons.HotReload">-->
+        <!--<keyboard-shortcut keymap="$default" first-keystroke="ctrl alt SEMICOLON"/>-->
+        <!--<keyboard-shortcut keymap="$default" first-keystroke="ctrl BACK_SLASH"/>-->
+      <!--</action>-->
```
Comment (pq) [inline resources/META-INF/plugin.xml]: Hmmm.  We do want it in both places though right?  In that case, can we replace one with an action reference?

    <reference ref="Flutter.Toolbar.ReloadAction"/>

Comment (devoncarew) [inline resources/META-INF/plugin.xml]: +1 to the action reference

**Comment by devoncarew** [comment]
@skybrian, sorry for the breakage! +1 to Phil's suggestion about using an action reference. If that doesn't work, please merge this and I'll look at alternative ways to have the action exposed in both places.

**Comment by skybrian** [comment]
Using a reference fixed it.

**Pull request merged**